src/components/AudioCell.vue

```diff
@@ -436,33 +436,66 @@ const moveDownCell = moveCell(1);
 // 消去
 const willRemove = ref(false);
 const removeCell = async () => {
-  // 1つだけの時は削除せず
-  if (audioKeys.value.length > 1) {
+  let audioKeysToDelete: AudioKey[];
+  if (
+    isMultiSelectEnabled.value &&
+    store.getters.SELECTED_AUDIO_KEYS.includes(props.audioKey)
+  ) {
+    audioKeysToDelete = store.state.audioKeys.filter((audioKey) =>
+      store.getters.SELECTED_AUDIO_KEYS.includes(audioKey)
+    );
```

Comment:
すみません、ここそもそもゲッター内でfilterされてそうでした 🙇 
https://github.com/sevenc-nanashi/voicevox/blob/1104de876026e225a654d0850b5a9f3864373663/src/store/audio.ts#L278-L282
```suggestion
    audioKeysToDelete = store.getters.SELECTED_AUDIO_KEYS;
```